fieldmanager.php

```diff
@@ -499,6 +499,23 @@ function fm_sanitize_textarea( $value ) {
 }
 
 /**
+ * Call wp_die() with an error message for Fieldmanager_Field::_failed_validation().
+ *
+ * @since 1.1.0
+ *
+ * @param string $error_message Optional. Error message in addition to the
+ *                              default message for validation errors.
+ */
+function fm_failed_validation_to_wp_die( $error_message = '' ) {
+	wp_die( esc_html(
+		$error_message
+		. "\n\n"
+		. __( "You may be able to use your browser's back button to resolve this error.", 'fieldmanager' )
+	) );
+}
+add_action( 'fm_failed_validation', 'fm_failed_validation_to_wp_die' );
```

Comment:
Quick important note, pressing back in the browser to get back to editing does not actually work, there should probably be a link to the post edit page instead, to have consistent experience across browsers and different cases.

And (2) is what I actually opted for when I tried to solve this solution. I'm totally against dying, and even exceptions, IMHO exceptions should only be thrown for invalid code, not invalid user input, and that includes debug mode as well. Visual notices like admin messages are better suited to inform users of their erroneous input.